tests/integration/test_social_media_services.py

```diff
@@ -0,0 +1,118 @@
+# Standard Library
+import json
+from unittest import mock
+
+# Third Party Stuff
+import pytest
+import tweepy
+from django.conf import settings
+from django.core.files.uploadedfile import SimpleUploadedFile
+from django.urls import reverse
+from tests import factories as f
+from tests import utils as u
+
+# nexus Stuff
+from nexus.base import exceptions as exc
+from nexus.social_media import services, tasks
+from nexus.social_media.models import Post
+
+pytestmark = pytest.mark.django_db
+
+
+@mock.patch('nexus.social_media.services.tweepy.API')
+@mock.patch('nexus.social_media.services.tweepy.OAuthHandler.set_access_token')
+@mock.patch('nexus.social_media.services.tweepy.OAuthHandler')
+def test_get_twitter_api_object(mock_oauthhandler, mock_set_access_token, mock_api):
+    mock_oauthhandler.return_value = tweepy.OAuthHandler
+    services.get_twitter_api_object(settings.TWITTER_OAUTH)
+
+    mock_oauthhandler.assert_called_once_with(
+        settings.TWITTER_OAUTH['consumer_key'], settings.TWITTER_OAUTH['consumer_secret']
+    )
+    mock_set_access_token.assert_called_once_with(
+        settings.TWITTER_OAUTH['access_key'], settings.TWITTER_OAUTH['access_secret']
+    )
+    mock_api.assert_called_once_with(mock_oauthhandler)
+
+    mock_api.side_effect = tweepy.error.TweepError('TweepError: Invalid Twitter OAuth Token(s).')
+    try:
+        services.get_twitter_api_object(settings.TWITTER_OAUTH)
+        assert False is True
```

Comment:
Please use `side_effects` in your mock object to test this scenario.

Don't play with boolean. It will make the logic in tests cryptic.